package.json

```diff
@@ -23,7 +23,7 @@
     "build": "vite build",
     "build-tokens": "node design-tokens/build.js",
     "build-storybook": "vite build & storybook build",
-    "intl": "formatjs extract src/components/**/*.ts*",
+    "intl-keys": "formatjs extract src/components/**/*.ts* --out-file i18n/keys.json",
```

Comment:
Should we make it part of the build command?

Reply:
Can be another ticket then

Reply:
I considered it, and I think the `releasability check` is better